docs/tutorials/image_classification/beginner.md

```diff
@@ -63,36 +42,63 @@ The best model is finally retrained on our entire dataset (i.e., merging trainin
 The best Top-1 accuracy achieved on the validation set is as follows:
 
 ```{.python .input}
-print('Top-1 val acc: %.3f' % classifier.results['best_reward'])
+fit_result = task.fit_summary()
+print('Top-1 train acc: %.3f, val acc: %.3f' %(fit_result['train_acc'], fit_result['valid_acc']))
 ```
 
 ## Predict on a New Image
 
-Given an example image, we can easily use the final model to `predict` the label (and the conditional class-probability):
+Given an example image, we can easily use the final model to `predict` the label (and the conditional class-probability denoted as `score`):
+
+```{.python .input}
+image_path = test_dataset.iloc[0]['image']
+result = classifier.predict(image_path)
+
+print(result)
+```
 
+You can also feed in multiple images all together, let's use images in test dataset as an example:
 ```{.python .input}
-# skip this if training FashionMNIST on CPU.
-if ag.get_gpu_count() > 0:
-    image = 'data/test/BabyShirt/BabyShirt_323.jpg'
-    ind, prob, _ = classifier.predict(image, plot=True)
+bulk_result = classifier.predict(test_dataset)
```

Comment:
Looking at the tutorial, I notice that the batch predict output differs from tabular, it looks like you have 1 row per class per image. Is this what users ideally want? If they wanted this format, wouldn't it make more sense to be in predict_proba, with N columns for N classes, each with the proba (as in Tabular)? This would keep the rows equal to the number of images, and allow for the index to remain aligned.

Would there be any drawbacks to having predict simply return the name of the majority class (as done in Tabular)?

Reply:
Okay, I will try keep it aligned with tabular predictor, for single input, return the category id; for multiple inputs, 1 row per image. The current `predict` is actually `predict_proba` in the context of tabular predictor.

Reply:
*correction, for `predict_proba` it still needs to group results by image